client/components/gsuite/gsuite-features/index.jsx

```diff
@@ -80,14 +80,12 @@ const GSuiteFeatures = ( { compact, domainName, productSlug, type } ) => {
 			/>
 			<GSuiteSingleFeature
 				title={
-					compact ? translate( 'Video and voice calls' ) : translate( 'Connect with your team' )
+					compact ? translate( 'Video calls' ) : translate( 'Connect with your team' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.